src/js/binaryen.idl

```diff
@@ -1,9 +1,38 @@
+enum WasmType {
```

Comment:
It's used in the Literal interface definition, exposing the `type` instance variable. So that the JS application using binaryen.js can find out the Literal type by calling `.get_type()` on a Literal instance.